.github/workflows/tests.yml

```
@@ -4,37 +4,58 @@ name: Continuous integration
 
 jobs:
   check:
-    name: Check
+    name: Check features
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           persist-credentials: false
-      - name: Rustup
-        run: |
-          rustup install --profile minimal stable
-          rustup override set stable
+      - uses: dtolnay/rust-toolchain@38b70195107dddab2c7bbd522bcf763bac00963b # stable
       - uses: Swatinem/rust-cache@f0deed1e0edfc6a9be95417288c0e1099b1eeec3 # v2.7.7
+      - uses: taiki-e/install-action@99b53876daf89877c0c7291dfd5033c8c0988612 # cargo-hack
```

nitpick, I prefer to be explicit about the version of the `install-action`:

```suggestion
      - uses: taiki-e/install-action@7ea888af71a31437ad4e71c62d021b3bb2728ea9 # v2.49.3
        with:
          tool: cargo-hack
```

I hope I got the indentation right, because the suggest-mode isn't super helpful :sweat_smile:

---

done.